rpmlb/cli.py

```diff
@@ -126,3 +134,11 @@ def run(recipe_file, collection_id, **option_dict):
     builder.run(work, **option_dict)
 
     log.info('Success!')
+
+
+def validate_dist(dist):
+    pattern = re.compile(DIST_RE)
```

Comment:
`re.compile` within function seems pointless – you will compile the pattern *again* each time the function is called. Compile once, on the top of the file:
```python
DIST_RE = re.compile('^(fc|el|centos)\d*$')
```
and the use the result many times:
```python
if ... and not DIST_RE.match(dist):
    ...
```

And if you need to access the original text pattern:
```python
'dist should match pattern {0}'.format(DIST_RE.pattern)
```